trio/tests/test_threads.py

```diff
@@ -457,6 +457,15 @@ def thread_fn():
     assert callee_token == caller_token
 
 
+async def test_trio_to_thread_run_sync():
```

Comment:
Can you please rename this to something like `test_trio_to_thread_run_sync_expected_error` or merge with the previous test?